src/experimental/mod.rs

```diff
@@ -0,0 +1,11 @@
+pub mod sbd;
+pub mod sbd_stat;
+pub mod stick_breaking;
+pub mod stick_breaking_stat;
+pub mod stick_sequence;
```

Comment:
Can you expose `Sb` from `sbd_prior` as well?

Reply:
`sbd_prior` was dead code, deleted it